app/models/admin.rb

```diff
@@ -1,2 +1,3 @@
 class Admin < ActiveRecord::Base
+    validates :uin, presence: true
```

Comment:
Can also add the validation for UIN if needed.
validates_format_of :uin, :with => /^\d+$/, :multiline => true
May not be necessary.